<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlanTaskWidget</class>
 <widget class="QWidget" name="PlanTaskWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>请输入查找线路：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="searchLineEdit">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="searchButton">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>查找</string>
           </property>
           <property name="icon">
            <iconset resource="../../../lz_master_main/resource/gui/icons.qrc">
             <normaloff>:/image/search.png</normaloff>:/image/search.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="showAllLinesButton">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>显示全部</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QListView" name="linesView">
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTableView" name="tunnelsView">
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="addTaskTunnelButton">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>加入</string>
         </property>
         <property name="icon">
          <iconset resource="../../../lz_master_main/resource/gui/icons.qrc">
           <normaloff>:/image/add.png</normaloff>:/image/add.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeTaskTunnelButton">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>删除</string>
         </property>
         <property name="icon">
          <iconset resource="../../../lz_master_main/resource/gui/icons.qrc">
           <normaloff>:/image/minus (2).png</normaloff>:/image/minus (2).png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addAllTaskTunnelsButton">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>全部加入</string>
         </property>
         <property name="icon">
          <iconset resource="../../../lz_master_main/resource/gui/icons.qrc">
           <normaloff>:/image/add.png</normaloff>:/image/add.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeAllTaskTunnelsButton">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>全部删除</string>
         </property>
         <property name="icon">
          <iconset resource="../../../lz_master_main/resource/gui/icons.qrc">
           <normaloff>:/image/minus white.png</normaloff>:/image/minus white.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="clearbutton">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>清空</string>
         </property>
         <property name="icon">
          <iconset resource="../../../lz_master_main/resource/gui/icons.qrc">
           <normaloff>:/image/reset.png</normaloff>:/image/reset.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>采集日期</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="dateLabel">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_10">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>车厢方向与出表方向</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QRadioButton" name="radioButton_5">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>一致    （同向）</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_6">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>不一致（反向）</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_11">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>相邻两帧间隔距离</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="pulseNumEdit">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhNone</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>默认0.5米/帧档帧间距</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QComboBox" name="comboBox">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <item>
               <property name="text">
                <string>0.5m</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>0.75m</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1m</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1.25m</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="actualPulseNum">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>m</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
           <zorder>actualPulseNum</zorder>
           <zorder>actualPulseNum</zorder>
           <zorder>comboBox</zorder>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QTableWidget" name="taskTunnelTableWidget">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="columnCount">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="nextButton">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>下一个</string>
           </property>
           <property name="icon">
            <iconset resource="../../../lz_master_main/resource/gui/icons.qrc">
             <normaloff>:/image/downloadstatus.png</normaloff>:/image/downloadstatus.png</iconset>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="turnOrderButton">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>全部翻转</string>
           </property>
           <property name="icon">
            <iconset resource="../../../lz_master_main/resource/gui/icons.qrc">
             <normaloff>:/image/downuploadstatus.png</normaloff>:/image/downuploadstatus.png</iconset>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="previousButton">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>上一个</string>
           </property>
           <property name="icon">
            <iconset resource="../../../lz_master_main/resource/gui/icons.qrc">
             <normaloff>:/image/uploadstatus.png</normaloff>:/image/uploadstatus.png</iconset>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="makeTaskFileButton">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>生成任务</string>
           </property>
           <property name="icon">
            <iconset resource="../../../lz_master_main/resource/gui/icons.qrc">
             <normaloff>:/image/filesave.png</normaloff>:/image/filesave.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>状态提示：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../lz_master_main/resource/gui/icons.qrc"/>
 </resources>
 <connections/>
</ui>
